Makefile

```diff
@@ -62,6 +64,11 @@ lint:
 release-major release-minor release-patch:
 	@$(XYZ) --increment $(@:release-%=%)
 
+.PHONY: bundle
+bundle:
+	mkdir dist || echo "Directory already exists"
+	$(BROWSERIFY) --standalone S index.js > dist/sanctuary.js
+	cat dist/sanctuary.js | $(UGLIFYJS) -mc > dist/sanctuary.min.js
```

Comment:
Here's the idiomatic way to express the dependencies:

```make
.PHONY: bundle
bundle: dist/sanctuary.js dist/sanctuary.min.js

.PHONY: dist/sanctuary.js
dist/sanctuary.js:
	mkdir -p -- '$(@D)'
	$(BROWSERIFY) --standalone S index.js >'$@'

dist/sanctuary.min.js: dist/sanctuary.js
	mkdir -p -- '$(@D)'
	cat '$<' | $(UGLIFYJS) -mc >'$@'
```

We could hard-code the dependencies of `dist/sanctuary.js` to avoid making it a phony target.